remote-api/src/test/java/org/alfresco/rest/api/tests/TestPeople.java

```diff
@@ -2371,4 +2372,21 @@ public String getScope()
     {
         return "public";
     }
+    
+    private ExpectedComparison expectedFirstName(Person person)
+    {
+        return new ExpectedComparison()
+        {
+            @Override
+            public void expected(Object other)
+            {
+                if (other instanceof Person) {
+                    assertEquals("firstName", person.getFirstName(), ((Person)other).getFirstName());
+                } else {
+                    fail("Not a person? "+other);
+                }

```

Comment:
The open brackets and the else keyword should all be on their own on a single as per Alfresco's formatting standards.